content/about/beaverbarcamp.rst

```diff
@@ -0,0 +1,26 @@
+Beaver BarCamp
+==============
+:slug: about/beaverbarcamp.rst
+:title: Beaver BarCamp
+:menu: Student Experience, Beaver BarCamp, 10
+
+.. image:: /images/barcamp.png
+    :scale: 100%
+    :align: center
+    :alt: Google Summer of Code
```

Comment:
Can we make this image a little smaller and maybe have it aligned on the right side so that the text wraps on the left? It just seems a little big and below the fold when you go to the page initially.